src/main/java/com/spotinst/sdkjava/model/SpotinstOceanGKELaunchSpecRepo.java

```diff
@@ -0,0 +1,79 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.exception.ExceptionHelper;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.gcp.OceanGKELaunchSpec;
+
+public class SpotinstOceanGKELaunchSpecRepo implements ISpotinstOceanGKECreateLaunchSpecRepo {
```

Comment:
Remove 'inst' from class name.

Reply:
Following naming convention of the existing classes. Do you still want to make changes